docs/latest/concepts/middleware.md

```diff
@@ -18,15 +18,15 @@ properties, e.g. `ctx.state.loggedIn = true`, but you can also replace the
 entire object like `ctx.state = { loggedIn: true }`.
 
 ```ts routes/_middleware.ts
-import { MiddlewareHandlerContext } from "$fresh/server.ts";
+import { FreshContext } from "$fresh/server.ts";
 
 interface State {
   data: string;
 }
 
 export async function handler(
   req: Request,
-  ctx: MiddlewareHandlerContext<State>,
+  ctx: FreshContext<State>,
 ) {
   ctx.state.data = "myData";
```

Comment:
The release documentation mentions:

`RouteContext (for async routes/layouts/app wrapper)` 

When I tried to confirm this, and use that typing in this code example, I got a typing error:

`Property 'next' does not exist on type 'RouteContext<State>'.deno-ts(2339)`